test/unit/ongoing-airdrops.spec.ts

```diff
@@ -204,6 +206,189 @@ describe('OngoingAirdrops', () => {
+      const claimees = [generateRandomAddress(), generateRandomAddress(), generateRandomAddress()];
+      const allocations = [
+        [10, 10],
+        [0, 5],
+        [50, 23],
+      ];
+      given(async () => {
+        campaignTokens = tokens.slice(0, 2);
+        ({ claimeesAllocations, tree, leaves } = await updateCampaign({
+          campaign,
+          claimees,
+          tokens: campaignTokens,
+          allocations,
+        }));
+      });
+      context('and claimee had already claimed', () => {
+        given(async () => {
+          await ongoingAirdrops.internalClaim(
+            {
+              campaign,
+              claimee: claimees[0],
+              recipient: generateRandomAddress(),
+            },
+            claimeesAllocations[0],
+            tree.getHexProof(leaves[0])
+          );
+        });
+        then('tx is reverted with custom error', async () => {
+          await expect(
+            ongoingAirdrops.internalClaim(
+              {
+                campaign,
+                claimee: claimees[0],
+                recipient: generateRandomAddress(),
+              },
+              claimeesAllocations[0],
+              tree.getHexProof(leaves[0])
+            )
+          ).to.be.revertedWithCustomError(ongoingAirdrops, 'AlreadyClaimed');
+        });
+      });
+      context('and claimee has something to claim', () => {
+        let claimTx: TransactionResponse;
+        const RECIPIENT = generateRandomAddress();
+        given(async () => {
+          claimTx = await ongoingAirdrops.internalClaim(
+            {
+              campaign,
+              claimee: claimees[0],
+              recipient: RECIPIENT,
+            },
+            claimeesAllocations[0],
+            tree.getHexProof(leaves[0])
+          );
+        });
+        then('total amount claimed by campaign, token and claimee is updated', async () => {
+          for (let i = 0; i < campaignTokens.length; i++) {
+            expect(
+              await ongoingAirdrops.amountClaimedByCampaignTokenAndClaimee(
+                getIdOfCampaignTokenAndClaimee(campaign, campaignTokens[i].address, claimees[0])
+              )
+            ).to.be.equal(claimeesAllocations[0][i].amount);
+          }
+        });
+        then('total claimed by campaign and token is updated', async () => {
+          for (let i = 0; i < campaignTokens.length; i++) {
+            expect(
+              await ongoingAirdrops.totalClaimedByCampaignAndToken(getIdOfCampaignAndToken(campaign, campaignTokens[i].address))
+            ).to.be.equal(claimeesAllocations[0][i].amount);
+          }
+        });
+        then('sends correct amount of tokens to recipient', async () => {
+          for (let i = 0; i < campaignTokens.length; i++) {
+            const [to, amount] = campaignTokens[i].transfer.getCall(0).args;
+            expect(to).to.be.equal(RECIPIENT);
+            expect(amount).to.be.equal(claimeesAllocations[0][i].amount);
```

Comment:
Not sure if it's the same or not, just a suggestion
```suggestion
            expect(campaignTokens[i].transfer).toHaveBeenCalledOnceWith(RECIPIENT, claimeesAllocations[0][i].amount)l
```